bin/reth/src/args/payload_builder_args.rs

```diff
@@ -36,6 +36,18 @@ pub struct PayloadBuilderArgs {
     /// Maximum number of tasks to spawn for building a payload.
     #[arg(long = "builder.max-tasks", help_heading = "Builder", default_value = "3", value_parser = RangedU64ValueParser::<usize>::new().range(1..))]
     pub max_payload_tasks: usize,
+
+    /// By default the pending block equals the latest block
+    /// to save resources and not leak txs from the tx-pool,
+    /// this flag enables computing of the pending block
+    /// from the tx-pool instead.
+    ///
+    /// If `compute_pending_block` is not enabled, the payload builder
+    /// will use the payload attributes from the latest block. Note
+    /// that this flag is not yet functional.
+    #[cfg(feature = "optimism")]
+    #[arg(long = "rollup.compute-pending-block")]
+    pub compute_pending_block: bool,
```

Comment:
wdym "not yet functional"?

Reply:
As in, not yet implemented. For context - the `compute_pending_block` flag was added to `op-geth` around the time that Lattice was making OP Craft. This flag is not necessary for MVP, so we decided to stub it out for now. 

In practice, the `compute_pending_block` flag is meant to allow for skipping the pending block computation in the payload job poller. See: https://github.com/ethereum-optimism/op-geth/blob/7b2e04673aaca3c33c78165d28089b4f5c456417/miner/worker.go#L429-L440